lib/HeroBonus.cpp

```diff
@@ -1095,6 +1117,60 @@ std::string Bonus::Description() const
 	return str.str();
 }
 
+JsonNode subtypeToJson(Bonus::BonusType type, int subtype)
+{
+	switch(type)

```

Comment:
We should find more generic solution for encoding sybtype and addInfo

Reply:
Agreed. But right now it's only used for converting legacy format (and still subject to manual copy-paste), so didn't want to cram a bigger change into this branch.